packages/react-charting/docs/TestPlans/AreaChart/ComponentTests.md

```diff
@@ -0,0 +1,60 @@
+|                 - Area chart mounted with empty data                  |                                       Empty chart aria label div rendered                                       |    Enzyme     |
+|                   Test 6: [Sub-Component]: Legends                    |                                                                                                                 |               |
+|                    - Hover mouse over area legends                    |                                  Should reduce the opacity of the other Areas                                   |      RTL      |
+|                    - Hover mouse over area legends                    | Should reduce the opacity of the other lines in area chart and opacity of the selected area line should be zero |      RTL      |
+|                    - Hover mouse over area legends                    |                                 Should reduce the opacity of the other legends                                  |      RTL      |
+|                    - Single mouse click on legends                    |                         Should select legend on single mouse click on respective legend                         |      RTL      |
+|                    - Double mouse click on legends                    |                        Should deselect legend on double mouse click on respective legend                        |      RTL      |
+|                Test 7: [Sub-Component]: x-axis labels                 |                                                                                                                 |               |
+|                       - Truncate x-axis labels                        |                               Should show the x-axis labels tooltip when hovered                                |      RTL      |
+|                        - Rotate x-axis labels                         |                                  Should rotate the x-axis labels by 39 degrees                                  |      RTL      |
+|              Test 8: [Sub-Component]: Screen resolution               |                                                                                                                 |               |
+|              - Increase the screen resolution (zoom in)               |                                       Should remain unchanged on zoom in                                        |      RTL      |
+|              - Decrease the screen resolution (zoom out)              |                                       Should remain unchanged on zoom out                                       |      RTL      |
+|                 Test 10: Theme changed to Dark Theme                  |                                           Should reflect theme change                                           |      RTL      |
```

Comment:
Please review the Test case Numbers. For example: After test case 4, test case 6 is given, test case 9 is also absent.